tegra-fuse.c

```diff
@@ -12,8 +12,21 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <errno.h>
+#include <linux/version.h>
```

Comment:
You shouldn't try to use compile-time kernel version checks in userland code.  For one thing, the kernel headers used in the cross-build environment don't match the actual kernel version.

Reply:
Thanks for the comment. I will fix that